homeworks/src/ua/kstt/justymenko/module06/LW_292_MatchStrings.java

```diff
@@ -0,0 +1,29 @@
+package ua.kstt.justymenko.module06;
+
+/**
+ * Created by n1kropol on 5/13/17.
+ */
+public class LW_292_MatchStrings {
+    public static void main(String[] args) {
+        String myStr1 = "Cartoon";
+        String myStr2 = "Tomcat";
+        StringBuilder builder = new StringBuilder();
+        for (int i = 0; i < myStr1.length(); i++) {
+            char finder = myStr1.charAt(i);
+            char match;
+            boolean isFinderPresent = false;
+            for (int j = 0; j < myStr2.length(); j++) {
+                match = myStr2.charAt(j);
+                if (finder == match) {
+                    isFinderPresent = true;
+                    break;
+                }
+            }
+
+            if (!isFinderPresent)
+                builder.append(finder);
+        }
+        System.out.println(builder.toString());
+//        Crn
+    }
+}
```

Comment:
as an alternative to the existing solution, come up with an algorithm with a better time complexity using a suitable data structure